include/mxnet/lib_api.h

```diff
@@ -1271,7 +1271,17 @@ extern "C" {
     for (int i = 0; i < num_opts; i++) {
       opts[std::string(opt_keys[i])] = std::string(opt_vals[i]);
     }
-    return supportedOps(subgraph_json, num_ids, ids, opts);
+    // create array of bools for operator support
+    std::vector<bool> _ids(num_ids, false);
+    // call user's supportedOps function
+    MXReturnValue retval = supportedOps(subgraph_json, _ids, opts);
+    if (!retval) return retval;
+
+    // copy bools in ids to ints
+    for (int i = 0; i < num_ids; i++)
```

Comment:
Do we need to update ids if retval is false?

Reply:
technically no, but we would need to check. so might as well update anyway